src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1193,9 +1176,9 @@ because xbuild doesn't support framework reference assemblies.
 		<_PropertyCacheItems Include="AndroidCreatePackagePerAbi=$(AndroidCreatePackagePerAbi)" />
 		<_PropertyCacheItems Include="AndroidGenerateJniMarshalMethods=$(AndroidGenerateJniMarshalMethods)" />
 		<_PropertyCacheItems Include="OS=$(OS)" />
-		<_PropertyCacheItems Include="DesignTimeBuild=$(DesignTimeBuild)" />
```

Comment:
@garuma I think this change is what is breaking the designer tests. Many of the targets skip now, and they used to _not_ skip.

Do you know if there are some cases where `DesignTimeBuild` is `False` with the designer?

Reply:
Yeah we set it for all the targets we run, that's the rundown:

- `UpdateAndroidResources` w/ DesignTimeBuild=true and AndroidUseManagedDesignTimeResourceGenerator=false
- `PrepareResources;_GenerateCompileInputs` w/ BuildProjectReferences=false, ContinueOnError=true, DesignTimeBuild=true, DeferredBuildSupported=true
- `GetExtraLibraryLocationsTargetName` w/ DesignTimeBuild=true

Reply:
Do you think a test could be leaving it off somehow?

Let me try reverting this line and see what happens.